Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -73,18 +73,18 @@ public struct TestCommand: AsyncParsableCommand, HasTrackableParameters {
 
     @Option(
         name: [.long, .customShort("T")],
-        help: "Path where test result bundle will be saved."
+        help: "[Deprecated] Path where test result bundle will be saved."
```

Comment:
I'd be against deprecating this. We will need tight control over the result bundle path to upload it to Tuist and for that, I wouldn't rely on the passthrough arguments.